python/lsst/eotest/image_utils.py

```diff
@@ -220,6 +374,38 @@ def fits_median(files, hdu=2, fix=True):
 
     return median_image
 
+def stack(ims, statistic=afwMath.MEDIAN):
+    """Stacks a list of images based on a statistic."""
+    images = []
+    for image in ims:
+        images.append(image)       
+    summary = afwMath.statisticsStack(images, statistic)
+    return summary
+
+
+def superbias(files, overscan, imaging=None, dxmin=5, dxmax=2, bias_method='row', 
+               hdu=2, statistic=afwMath.MEDIAN, **kwargs):
+    """Generates a single stacked 'super' bias frame based on 
+    a statistic. Images must be either all masked or all unmasked."""
+    ims = [afwImage.ImageF(f, hdu) for f in files]
+    bias_frames = [unbias_and_trim(im, overscan, imaging, dxmin, dxmax, bias_method,
+                                   **kwargs) for im in ims]
+    return stack(bias_frames, statistic)
+
+def superbias_file(files, overscan, outfile, imaging=None, dxmin=5, dxmax=2, 
+                    bias_method='row', bitpix=16, clobber=True, **kwargs):
+    output = fits.open(files[0])
```

Comment:
Instead of using `fits.open` and setting the `.data` attributes of each hdu, I would create a dictionary of `afw_image`s like this:
```
images = {amp: super_bias(files, overscan, imaging, dxmin, dxmax, bias_method, hdu=dm_hdu(amp), **kwargs)
                  for amp in allAmps(files[0])}
imutils.writeFits(images, outfile, files[0], bitpix=bitpix)
```
The `writeFits` function is https://github.com/lsst-camera-dh/eotest/blob/master/python/lsst/eotest/image_utils.py#L224

Reply:
Thanks, I've made the change.

Reply:
The only issue I'm having now when I run `superbias_file()` is that I can't set `bitpix=None`, which I use to return float values for my superbias. This is because of a line in `writeFits`:

``` 
for amp in all_amps:
        if bitpix < 0:
            output.append(fits.ImageHDU(data=images[amp].getArray()))
```
where it can't evaluate `bitpix < 0` for `bitpix=None`.